src/main/java/org/roaringbitmap/RoaringArray.java

```diff
@@ -578,4 +578,30 @@ protected int size() {
   public void writeExternal(ObjectOutput out) throws IOException {
     serialize(out);
   }
+
+  /**
+   * Gets the first value in the array
+   * @return te first value in the array
+   */
+  public int first() {
+    if(size == 0) {
+      return 0;
```

Comment:
Is this the good policy to return 0 as the value when there is no elements in the RoaringArray (or a RoaringBitmap) ?
I thought that 0 would be a valid value. We may simply throw an error `NoSuchElementException`, as Collections.max does in the JDK.